gammapy/datasets/make.py

```diff
@@ -64,3 +72,147 @@ def sigma_energy_theta(energy, theta, sigma):
+    obs_table['TIME_OBSERVATION'] = time_observation
+
+    # livetime: 25 min
+    time_live = Quantity(25.*np.ones_like(obs_id), 'minute').to('second')
+    obs_table['TIME_LIVE'] = time_live
+
+    # start time
+    # random points between the start of 2010 and the end of 2014
+    # using the start of 2010 as a reference time for the header of the table
+    # observations restrict to night time
+    # considering start of astronomical day at midday: implicit in setting the
+    # start of the night, when generating random night hours
+    datestart = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    dateend = Time('2015-01-01 00:00:00', format='iso', scale='utc')
+    time_start = Time((dateend.mjd - datestart.mjd)
+                      *np.random.random(len(obs_id))
+                      + datestart.mjd, format='mjd', scale='utc')
+
+    # keep only the integer part (i.e. the day, not the fraction of the day)
+    time_start_f, time_start_i = np.modf(time_start.mjd)
+    time_start = Time(time_start_i, format='mjd', scale='utc')
+
+    # random generation of night hours: 6 h (from 22 h to 4 h), leaving 1/2 h
+    # time for the last run to finish
+    night_start = Quantity(22., 'hour')
+    night_duration = Quantity(5.5, 'hour')
+    hour_start = night_start + night_duration*np.random.random(len(obs_id))
+
+    # add night hour to integer part of MJD
+    time_start += hour_start
+
+    if debug :
+        # show the observation times in UTC
+        time_start = time_start.iso
+    else :
+        # show the observation times in seconds after the reference
+        time_start = time_relative_to_ref(time_start, header)
+        # converting to quantity (beter treatment of units)
+        time_start = Quantity(time_start.sec, 'second')
+
+    obs_table['TIME_START'] = time_start
+
+    # stop time
+    # calculated as TIME_START + TIME_OBSERVATION
+    if debug :
+        time_stop = Time(obs_table['TIME_START']) + TimeDelta(obs_table['TIME_OBSERVATION'])
+    else :
+        time_stop = TimeDelta(obs_table['TIME_START']) + TimeDelta(obs_table['TIME_OBSERVATION'])
+        # converting to quantity (beter treatment of units)
+        time_stop = Quantity(time_stop.sec, 'second')
+
+    obs_table['TIME_STOP'] = time_stop
+
+    # az, alt
+    # random points in a sphere above 45 deg altitude
+    az, alt = sample_sphere(len(obs_id), (0, 360), (45, 90), 'degree')
+    az = Angle(az, 'degree')
+    alt = Angle(alt, 'degree')
+    obs_table['AZ'] = az
+    obs_table['ALT'] = alt
+
+    # RA, dec
+    # derive from az, alt taking into account that alt, az represent the values
+    # at the middle of the observation, i.e. at time_ref + (TIME_START + TIME_STOP)/2
+    # (or better: time_ref + TIME_START + (TIME_OBSERVATION/2))
+    # in debug modus, the time_ref should not be added, since it's already included
+    # in TIME_START and TIME_STOP
+    az = Angle(obs_table['AZ'])
+    alt = Angle(obs_table['ALT'])
+    if debug :
```

Comment:
PEP8 will give a warning here, there should be no space before the colon:
`debug :` -> `debug:`

This is not very important, but please run `pep8` on this file to make sure your code formatting is OK.